### DISCUSSION TITLE
How to show label by default on map pin?

### DISCUSSION THREAD

**Discussion by intellimedhu** [dotnet/maui#11248] [Q&A]
1. Is it possible to show the label of a pin on the map control whithout tapping on it? Can the pin's touch event be raised from code?

2. If the text in the pin's label is too long it will be truncuated.  Is it possible the display longer texts (maybe with line breaking)?

**Comment by mprogers** [comment]
I have a student who is trying to do mapping, and was saying that pins aren't implemented for .NET 6 - are you working on .NET 7, or is my student just missing something?

**Reply by intellimedhu** [reply]
Yes, I'm working on .NET 7.
